src/pages/Templates/ZTemplatesPage.css

```diff
@@ -7,3 +7,14 @@
   padding-left: 94px;
```

Comment:
There is twice the same padding, maybe you can delete one of them? 
Here : 
```  
padding-left: 94px;
padding-right: 30px; 
```
and here : 
``` 
.flex-templates {
...
padding-left: 94px;
padding-right: 30px;
 } 
```